pkgs/development/lua-modules/overrides.nix

```diff
@@ -517,9 +517,14 @@ in
     meta.broken = stdenv.hostPlatform.isDarwin;
   });
 
-  lush-nvim = prev.lush-nvim.overrideAttrs (drv: {
+  lush-nvim = prev.lush-nvim.overrideAttrs {
     doCheck = false;
-  });
+    postInstall = ''
```

Comment:
I'm assuming we don't need to concat with `old.postInstall or ""` because this is overriding an auto-generated package?

Reply:
There's isn't an existing `postInstall`. Normal operations are done in the `installPhase` and you can hook additional behavior in `pre/post`.

Reply:
Sometimes I end up overriding a package that is either already overridden or uses a builder that encourages using the hooks in the base package impl.

My point was we can safely assume that that isn't the case here, which you've confirmed 👌